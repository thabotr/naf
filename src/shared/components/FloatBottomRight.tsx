import React from 'react';
import {StyleSheet} from 'react-native';
import {Surface} from 'react-native-paper';

type Props = any;

export default function (props: Props) {
  return <Surface style={[styles.floater, props.style]} {...props} />;
}

const styles = StyleSheet.create({
  floater: {
    position: 'absolute',
    right: 40,
    bottom: 50,
    elevation: 3,
  },
});
