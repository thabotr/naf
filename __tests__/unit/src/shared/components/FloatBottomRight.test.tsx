import React from 'react';
import renderer from 'react-test-renderer';
import FloatBottomRight from '../../../../../src/shared/components/FloatBottomRight';

describe('FloatBottomRight', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<FloatBottomRight />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
