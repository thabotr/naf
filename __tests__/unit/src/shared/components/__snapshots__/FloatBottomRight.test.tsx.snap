// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FloatBottomRight renders correctly 1`] = `
<View
  collapsable={false}
  style={
    {
      "backgroundColor": "#ffffff",
      "bottom": 50,
      "elevation": 3,
      "position": "absolute",
      "right": 40,
      "shadowColor": "#000000",
      "shadowOffset": {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.24,
      "shadowRadius": 3,
    }
  }
/>
`;
